python/lsst/ts/xml/data/sal_interfaces/ATMonochromator/ATMonochromator_Commands.xml

```diff
@@ -7,9 +7,9 @@
     <Description>Command to change the wavelength of the monochromator</Description>
     <item>
       <EFDB_Name>wavelength</EFDB_Name>
-      <Description>wavelength</Description>
+      <Description>wavelength to be calibrated</Description>
```

Comment:
Please to be consistent, start Description as sentence with capital - Wavelength

Reply:
And terminate the sentence in a `.`. Apply to all Description fields.

Reply:
Thanks for taking a look! I will clean it all up!